algebraic-graphs.cabal

```diff
@@ -104,7 +105,8 @@ library
                         base-compat >= 0.9.1   && < 0.11,
                         containers  >= 0.5.5.1 && < 0.8,
                         deepseq     >= 1.3.0.1 && < 1.5,
-                        mtl         >= 2.1     && < 2.3
+                        mtl         >= 2.1     && < 2.3,
+                        transformers
```

Comment:
It doesn't look like we actually need this package.

Reply:
Oh no, I've needed it previously but not anymore. I'll remove it in the next commit.